contracts/bonding/BondingManager.sol

```diff
@@ -850,28 +771,29 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
     }
 
     /**
-     * @dev Return total active stake for a round
-     * @param _round Round number
+     * @dev Return the round for which a transcoder's stake was last updated
+     * @param _transcoder Transcoder address
      */
-    function getTotalActiveStake(uint256 _round) public view returns (uint256) {
-        return activeTranscoderSet[_round].totalStake;
+    function lastActiveStakeUpdateRound(address _transcoder) public view returns (uint256) {
+        return transcoders[_transcoder].lastActiveStakeUpdateRound;
     }
 
-    /**
-     * @dev Return whether a transcoder was active during a round
+   /**
+     * @dev Return whether a transcoder is active for the current round
      * @param _transcoder Transcoder address
-     * @param _round Round number
      */
-    function isActiveTranscoder(address _transcoder, uint256 _round) public view returns (bool) {
-        return activeTranscoderSet[_round].isActive[_transcoder];
+    function isActiveTranscoder(address _transcoder) public view returns (bool) {
+        uint256 activationRound = transcoders[_transcoder].activationRound;
+        return activationRound != 0 && activationRound <= roundsManager().currentRound();
     }
 
     /**
      * @dev Return whether a transcoder is registered
      * @param _transcoder Transcoder address
      */
     function isRegisteredTranscoder(address _transcoder) public view returns (bool) {
-        return transcoderStatus(_transcoder) == TranscoderStatus.Registered;
```

Comment:
We can make the update to client code that depends on the BondingManager a bit smoother by keeping this definition and the `TranscoderStatus` enum.

Reply:
`isRegisteredTranscoder` is used externally as well in the `pm` contracts and needs to return a boolean.

The client facing method returning the enum values would be `transcoderStatus()` which has been removed as it is no longer used within the contracts itself, but can be re-added to be client-facing. 

Then again we'd only have to update the abigen generated function for `isRegisteredTranscoder` to return an enumerated value rather than a boolean value returned from the RPC call I think ? That way we wouldn't have to make additional adjustments. Downside here is that we'd have to re-add that code when we re-generate the client code for the bondingmanager using abigen.

Since `TranscoderStatus` only contains 2 possible values the return values from the RPC call might as well be boolean.

Reply:
We can avoid changes to `isRegisteredTranscoder()` all together s`transcoderStatus()` would only be a client-facing wrapper for the return values. WDYT?

```
function transcoderStatus(address _transcoder) returns (TranscoderStatus) {
   if (isRegisteredTranscoder(_transcoder) return TranscoderStatus.Registered;
  return TranscoderStatus.NotRegistered;
}
```